packages/codemirror-blocks/src/toolkit/test-utils.tsx

```diff
@@ -1,6 +1,6 @@
 import { API, CodeMirrorBlocksComponent, Language } from "../CodeMirrorBlocks";
 import { act, cleanup, render } from "@testing-library/react";
-import { afterAllDOMUpdates, cancelAllDOMUpdates } from "../utils";
+// import { afterAllDOMUpdates, cancelAllDOMUpdates } from "../utils";
```

Comment:
Any reason not to just remove this line entirely?

Reply:
oops, thanks!